src/server/helpers/validSchemes.js

```diff
@@ -0,0 +1,66 @@
+const Joi = require('joi');
+
+exports.participant = Joi.object().keys({
+  forename: Joi.string().min(1).max(30).required(),
+  surename: Joi.string().min(1).max(30).required(),
+  mobile: Joi.string().regex(/^\d+$/).required(),
```

Comment:
@knanahassouna1 Hey! I think this regex might need to be "relaxed" a little bit.  I think users are pretty inconsistent in how they enter their mobile numbers, and will often have spaces, dashes, periods, a plus sign (for country code), and so on:
5635908350
(563) 590-8350
+1 563 590 8350
563.590.8350
So it might be less painful for the user if you allow all of that, and then (if needed) you can scrub out the extra characters/symbols when storing in the database.